doc/releases/changelog-dev.md

```diff
@@ -180,6 +180,32 @@
   tensor([0.69301172, 0.67552491, 0.65128847], requires_grad=True)
   ```
 
+* Added new optimizer `qml.SPSAOptimizer` for noisy problems. It performs less device executions, due
+to the fact that it performs two measures to determine the gradient.
+[(#2661)](https://github.com/PennyLaneAI/pennylane/pull/2661)
+
+```python
+dev = qml.device("default.qubit", wires=1)
+def circuit(params, wires):
+    qml.BasisState(np.array([1, 1, 0, 0]), wires=wires)
+    for i in wires:
+        qml.Rot(*params[i], wires=i)
+    qml.CNOT(wires=[2, 3])
+    qml.CNOT(wires=[2, 0])
+    qml.CNOT(wires=[3, 1])
+
+def exp_val_circuit(params):
+    circuit(params, range(dev.num_wires))
+    return qml.expval(h2_ham)
```

Comment:
`h2_ham` is not defined - would be good to have a def for it such that the example is runnable simply (or the observable may be different).